### PR TITLE
deploy/charts: allow configuring otel-metrics

### PR DESCRIPTION
**Pull request by mqasimsarfraz**
Allow configuring otel-metrics operator

### CODE REVIEW

**Comment by flyth** [comment]
Do you also have plans regarding the [other exporters](https://www.inspektor-gadget.io/docs/v0.34.0/reference/export-metrics#other-exporters)?

File: cmd/kubectl-gadget/deploy.go [hunk 1]
```diff
@@ -258,6 +260,12 @@ func init() {
 	deployCmd.PersistentFlags().BoolVar(
 		&disallowGadgetsPull,
 		"disallow-gadgets-pulling", false, "Disallow pulling gadgets from registries")
+	deployCmd.PersistentFlags().BoolVar(
```
Comment (blanquicet) [inline cmd/kubectl-gadget/deploy.go:263]: Thanks for working on this!

Could you please update [exporting metrics documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/reference/export-metrics.mdx) and also [profile blockio documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/gadgets/profile_blockio/README.mdx#exporting-metrics)?

**Review by blanquicet — COMMENTED** [review]
LGTM! Just need to synch flag descriptions with `ig` and `gadgetctl`.

File: cmd/kubectl-gadget/deploy.go [hunk 2]
```diff
@@ -258,6 +260,12 @@ func init() {
...
+		&otelMetricsListen,
+		"otel-metrics-listen", false, "Enable OpenTelemetry metrics listener")
+	deployCmd.PersistentFlags().StringVar(
+		&otelMetricsListenAddr,
+		"otel-metrics-listen-address", "0.0.0.0:2224", "Address to listen for OpenTelemetry metrics")
```
[the remark this answers is not in the document]
Comment (blanquicet) [reply cmd/kubectl-gadget/deploy.go]: I fixed it myself to move this PR forward 🙂

**Comment by blanquicet** [comment]
Rebased to solve conflicts.

**Review by blanquicet — APPROVED** [review]
LGTM!

**Pull request merged**